cas-server-core-api-ticket/src/main/java/org/jasig/cas/ticket/registry/TicketRegistryState.java

```diff
@@ -15,7 +15,7 @@
      * @return Number of ticket-granting tickets in the registry at time of invocation
      *         or {@link Integer#MIN_VALUE} if unknown.
      */
-    int sessionCount();
+    long sessionCount();
```

Comment:
To expand on that last note: a remember-me policy typically indicates a timeout value of a very large value; much larger than int, because you want your session to last, let's say 3 weeks, etc. As long as that value converts to a long, CAS would bomb.  It's also not so much about the semantics of the type. The timeout values that we have for expiration policies of STs and TGTs all resolve to a long today. There is the conflict; the API (prior to this change) allows for a TGT timeout to be a long, yet the count of sessions expects an int. That does not make sense.